onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_RestrictedIssueTest.java

```diff
@@ -0,0 +1,269 @@
+public class ONPRC_RestrictedIssueTest extends BaseWebDriverTest implements SqlserverOnlyTest
+{
+    private final IssuesHelper _issuesHelper;
+
+    // constants
+    private static final TestUser USER1 = new TestUser("user1_issuetest@issues.test");
+    private static final TestUser USER2 = new TestUser("user2_issuetest@issues.test");
+    private static final TestUser ISSUE_CREATOR = new TestUser("issue_creator_issuetest@issues.test");
+    private static final TestUser FOLDER_ADMIN = new TestUser("folder_admin_issuetest@issues.test");
+
+    private static final String RESTRICTED_ISSUES_LIST = "restricted-issues";
+    private static final String UNRESTRICTED_ISSUES_LIST = "unrestricted-issues";
+    private static final String ACCESS_ERROR_MSG = "This issue is in a restricted issue list. You do not have access to this issue";
+
+    public ONPRC_RestrictedIssueTest()
+    {
+        _issuesHelper = new IssuesHelper(this);
+    }
+
+    @BeforeClass
+    public static void doSetup()
+    {
+        ONPRC_RestrictedIssueTest initTest = getCurrentTest();
+        initTest.doInit();
+    }
+
+    public void doInit()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+
+        // Create test users
+        USER1.create(this).addPermission("Editor", getProjectName());
+        USER2.create(this).addPermission("Editor", getProjectName());
+        ISSUE_CREATOR.create(this).addPermission("Editor", getProjectName());
+        FOLDER_ADMIN.create(this).addPermission("Folder Administrator", getProjectName());
+
+        // Add issue list definitions
+        _issuesHelper.createNewIssuesList(RESTRICTED_ISSUES_LIST, _containerHelper, true, false, false);
+        waitAndClickAndWait(Locator.linkContainingText(RESTRICTED_ISSUES_LIST));
+        _issuesHelper.goToAdmin();
+        _issuesHelper.setRestrictedIssueList(true);
+        _issuesHelper.setIssueAssignmentList("Site: Users");
+        clickButton("Save");
+
+        clickProject(getProjectName());
+        _issuesHelper.createNewIssuesList(UNRESTRICTED_ISSUES_LIST, _containerHelper, false, false, false);
+        waitAndClickAndWait(Locator.linkContainingText(UNRESTRICTED_ISSUES_LIST));
+        _issuesHelper.goToAdmin();
+        _issuesHelper.setIssueAssignmentList("Site: Users");
+        clickButton("Save");
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        _userHelper.deleteUsers(false, USER1, USER2, ISSUE_CREATOR, FOLDER_ADMIN);
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("ehr", "onprc_ehr");
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return "RestrictedIssuesVerifyProject";
+    }
+
+    @Test
+    public void restrictedIssueTest()
+    {
+        clickProject(getProjectName());
```

Comment:
goToProjectHome(); would do the same thing.

Reply:
I don't have a particular preference but the difference is that `clickProject` uses the project menu while `goToProjectHome` navigates directly to `<containerPath>/project-begin.view`.